doc/source/visualizing/plots.rst

```diff
@@ -295,7 +296,7 @@ example:
    ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
    prj = yt.ProjectionPlot(
        ds,
-       2,
+       "z",
```

Comment:
out of curiosity, how did you find those ?

Reply:
I noticed them while reading